snapshot-labs/snapshot
# feat: Separate networks

### Summary



Closes: https://github.com/snapshot-labs/pitches/issues/33

### How to test

1. Set `.env.local` to `demo`.
2. Choose a testnet and click "Save".
3. Change `.env.local` to `production`.
4. In settings, notice the testnet is still chosen but not searchable or selectable.
5. Click "Save" to confirm settings and observe the error message.


<img width="744" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/b91e6a81-1644-4988-9482-2579246cbac9">


### To-Do

- [ ] Figure out how we want to handle migration for users and what message to show for migration 
- [x] Waiting for https://github.com/snapshot-labs/snapshot.js/pull/936
- [x] Waiting for https://github.com/snapshot-labs/snapshot-sequencer/pull/239
- [x] Combobox: Fix hidden items and fix error prop
